### DISCUSSION TITLE
Add block to specified location in the page

### DISCUSSION THREAD

**Discussion by bergercookie** [ramnes/notion-sdk-py#96] [Q&A]
Hi there,

I'm aware that you can append blocks at the end of pages, by specifying something in the lines of:

```python
notion.blocks.children.append(block_id=page_id, children=[some_new_block])
```

is there a way of adding a block at the beginning or more generally at a specified location of the page instead?

**Comment by ramnes** [comment]
Well, you can append to a block in particular, for example a paragraph, a to do list, or a quote. Just replace `page_id` with the id of the block you want to append to. But I don't think there's a way to append at a specific location yet.
